CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ include(autogenerate_plugin_container.cmake)
 include_directories(
     include
     libs/include
+    libs/json11
```

Comment:
put `libs/json11` to the bottom of this list and put a `SYSTEM` in front of it. This way the warnings are ignored.